models/public/mozilla-deepspeech-0.8.2/mozilla-deepspeech-0.8.2.md

```diff
@@ -0,0 +1,138 @@
+# mozilla-deepspeech-0.8.2
+
+## Use Case and High-Level Description
+
+"mozilla-deepspeech-0.8.2" is a speech recognition neural network pre-trained by Mozilla
+based on DeepSpeech architecture (CTC decoder with beam search and n-gram language model)
+with changed neural network topology.
+
+For details on the original DeepSpeech, see paper <https://arxiv.org/abs/1412.5567>.
+
+For details on this model, see <https://github.com/mozilla/DeepSpeech/releases/tag/v0.8.2>.
+
+## Conversion
+
+1. Download `deepspeech-0.8.2-models.pbmm` and `deepspeech-0.8.2-models.scorer` files from Mozilla's repository (MPL-2.0 license):
+   ```
+   source <openvino_path>/bin/setupvars.sh
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/downloader.py --name mozilla-deepspeech-0.8.2
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mozilla-deepspeech-0.8.2
```

Comment:
This should go after conversions (if the conversion approach isn't changed).

Reply:
Oops, right. But I'm removing this section anyway.

Reply:
Done, no "Conversion" section in this file now.